internal/scripts/scripts.go

```diff
@@ -208,3 +197,23 @@ func (c *ContentValue) List(thread *starlark.Thread, fn *starlark.Builtin, args
 	}
 	return starlark.NewList(values), nil
 }
+
+func (c *ContentValue) Symlink(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (Value, error) {
+	var slTarget, slLinkPath starlark.String
+	err := starlark.UnpackArgs("Content.symlink", args, kwargs, "target", &slTarget, "linkpath", &slLinkPath)
+	if err != nil {
+		return nil, err
+	}
+
+	target := slTarget.GoString()
+	linkPath := slLinkPath.GoString()
+	linkPath, err = c.RealPath(linkPath, nil)
```

Comment:
Why was the containment check disabled here?

Reply:
The comment refers to outdated check, but the containment check for linkPath is still not present because there's no way to specify symlink to be created in slice YAML. Currently you can only specify mutable plain files. (With non-empty `text`. This is fixed in other PR.) Do you have any suggestion how symlinks created by scripts should be specified?